src/main/java/gift/controller/ProductController.java

```diff
@@ -0,0 +1,60 @@
+package gift.controller;
+
+import gift.dto.ProductRequestDto;
+import gift.dto.ProductResponseDto;
+import gift.service.ProductServiceImpl;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/products")
+public class ProductController {
+
+    private final ProductServiceImpl productService;
+
+    public ProductController(ProductServiceImpl productService) {
+        this.productService = productService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<ProductResponseDto>> findAllProducts() {
+        List<ProductResponseDto> productList = productService.findAllProducts();
+
+        return new ResponseEntity<>(productList, HttpStatus.OK);
```

Comment:
```suggestion
        return ResponseEntity.ok(productList);
```
이런식의 정적 팩토리 메서드를 사용할 수도 있습니다. 물론 지금 코드도 좋아요👍👍